File: 8th kyu/How good are you really?.js
// Solution
function betterThanAverage(classPoints, yourPoints) {
  // Your code here
  let sum = 0
  let tick = 0
  let average = 0
  for (let i = 0; i < classPoints.length; i++) {
    sum += classPoints[i]
    tick += 1
    average = sum/tick
    console.log(average)
  }
  if ( yourPoints > average) {
    return true
  } else {
    return false
  }

}

File: 8th kyu/How good are you really?/how_good_are_you_really?.py
# Solution 1
def better_than_average(class_points, your_points):
    denominator = 0
    for point in class_points:
        denominator = denominator + 1
        
    average = (sum(class_points) / denominator)
    
    if average > your_points:
        return False
    else:
        return True